services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/logging/YhteystietoLoggingEntryHolder.kt

```diff
@@ -28,6 +28,10 @@ const val PERSONAL_DATA_LOGGING_MAX_IP_LENGTH = 40
  */
 class YhteystietoLoggingEntryHolder {
 
+    // To Constants?
```

Comment:
Huh, this bring back, partially, old version... Something went wrong with merging/rebasing from latest dev?